### DISCUSSION TITLE
Problem when update moose/conda

### DISCUSSION THREAD

**Discussion by aaelmeli** [idaholab/moose#23727] [Q&A General] [answered]
Hi

I have updated moose repo and conda but when I rebuild moose, i get this error

```
In file included from /home/aaelmeli/projects/moose/framework/build/unity_src/meshgenerators_Unity.C:65:
/home/aaelmeli/projects/moose/framework/src/meshgenerators/SymmetryTransformGenerator.C: In member function 'virtual std::unique_ptr<libMesh::MeshBase> SymmetryTransformGenerator::generate()':
/home/aaelmeli/projects/moose/framework/src/meshgenerators/SymmetryTransformGenerator.C:90:28: error: 'orient_elements' is not a member of 'libMesh::MeshTools::Modification'
   90 |   MeshTools::Modification::orient_elements(*mesh);
      |                            ^~~~~~~~~~~~~~~
```

Do I have to remove the entire moose and conda and installed it again from scratch? any other way?

If yes, is there any other way to solve this for next updates? I experienced this couple of times and it is somewhat frustrating to reinstall moose and conda at every moose update.

Thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

This means you need to update libmesh.
The conda update must have failed if you did that.

You may remove the conda environment and install it again
You do not need to remove moose, you ll just need to install it again after updating libmesh

Guillaume

**Reply by aaelmeli** [reply]
So, do I only need to perform these commands?

```
mamba activate base
mamba env remove -n moose
```

then reinstall it again by using

`mamba create -n moose moose-tools moose-libmesh`

?

**Reply by GiudGiud** [reply]
yes this should work
